template/sbt.go

```diff
@@ -13,7 +13,7 @@ sbtscratch:
     {{.ScratchVolumes}}
 {{end}}
 sbt: &sbt
-  image: arch-docker.eng.lancope.local:5000/sbt
+  image: test.local:5000/sbt
```

Comment:
This needs to be changed to an image that is available in Docker Hub. Setting it to test.local:5000/sbt will fail when run.